catfs/fs.go

```diff
@@ -32,9 +32,14 @@ import (
 )
 
 const (
-	abiVersion = 1
+	abiVersion                 = 1
+	defaultEncryptionKeyLength = 32
 )
 
+func defaultEncryptionKey() []byte {
```

Comment:
The name is maybe a bit misleading: It's not the default, nor the initial one. It's just the one for empty files.
Mentioning, because it would ring my alarms when I'd read this in a foreign code base.

Reply:
Indeed, I was not clear enough. In 0776b1e8 I changed
`defaultEncryptionKey` to  `emptyFileEncryptionKey`